proto/utxorpc/v1alpha/cardano/cardano.proto

```diff
@@ -596,6 +596,24 @@ message PParams {
   uint64 drep_inactivity_period = 31; // The drep inactivity period.
 }
 
+message EraBoundary {
+  uint64 time = 1; // ms timestamp
+  uint64 slot = 2; // absolute slot number of the first block of this era
+  uint64 epoch = 3; // first epoch for this era
+  optional bytes hash = 4; // block hash of the first block of this era (if the era has started)
+}
+message EraSummary {
+  // TODO: do we want any kind of "era name" identifier?
+  //       a lot of tools expect a string name (ex: "shelley") to look up era info
```

Comment:
My feeling is we should probably add this, but open to thoughts.

The rationale is that the `EraSummary` endpoint does not actually tell you the protocol parameters for this era. Rather, you have to query a node (or read from a disk) the right genesis file to get the full data. However, these fields & endpoints use a string name (ex: `shelley`) as the identifier, so having a string here would help fetch this info

The alternative, if there is no name, is that clients would have to hardcode it (ex: the first "summary" is for "shelley"), but this is boilerplate and also error-prone (you have to remember to update this mapping every hardfork)